Style.css

```diff
@@ -325,3 +345,308 @@ button:disabled {
   width: 100px;
   border-radius: 10px;
 }
+
+@media (min-width: 768px) {
+  .img1 {
+    content: url("./desktop-images/Snapshoot\ Portfolio.png");
+  }
+
+  .img2 {
+    content: url("./desktop-images/Snapshoot\ Portfolio-1.png");
+  }
+
+  .img3 {
+    content: url("./desktop-images/Snapshoot\ Portfolio-2.png");
+  }
+
+  .img4 {
+    content: url("./desktop-images/Snapshoot\ Portfolio-3.png");
+  }
+
+  .wrapper {
+    border: none;
+    width: 128%;
+    background-color: #f4f5f7;
+  }
+
+  header {
+    width: 100%;
+    position: fixed;
+    top: -3vh;
+    background: white;
+    z-index: 1;
+  }
+
+  .navbar {
+    margin-top: -3%;
+  }
+
+  .logo {
+    align-self: center;
+  }
+
+  .union {
+    display: none;
+  }
+
+  .page-flow {
+    display: flex;
+    min-width: 35%;
+    justify-content: space-between;
+  }
+
+  .page-flow > li > a {
+    text-decoration: none;
+  }
+
+  .page-flow > li > a:hover {
+    color: blue;
+  }
+
+  .headline {
+    margin-top: 5.5%;
+    background: url(./desktop-images/Header\ bg.png);
+    background-size: cover;
+    background-repeat: no-repeat;
+    padding: 70px 200px;
+    background-color: white;
+    height: 100vh;
+  }
+
+  h1 {
+    font-size: 48px;
+    line-height: 60px;
+    letter-spacing: 0.37px;
+  }
+
+  .body-text {
+    font-size: 20px;
+    line-height: 28px;
+  }
+
+  .works {
+    padding: 0 5%;
+  }
```

Comment:
- [ ] Please kindly re-check, I noticed that all your cards are not responsive as expected; @ min-width: 768px this is what I have 👇 (The same thing apply to `.headline` and `.contact-me` sections as well).

## 
![socialmedia](https://user-images.githubusercontent.com/103386183/206439491-54c775f9-c271-444d-a76f-bf9b2b9f11e3.png)

## 
![socialmedia](https://user-images.githubusercontent.com/103386183/206441136-100cc2a7-24f3-4821-ad06-cd06f004b4fe.png)